src/applications/disability-benefits/2346/Components/OrderCommentPage.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+const OrderCommentPage = () => {
+  const [checked, setChecked] = useState(false);
```

Comment:
Why does this component use a ```useState``` hook pattern yet the component above is using connect?  We should probably be consistent with how we're accessing state.

Reply:
I think using `useState` here is probably ok, but you'll need to pass in a function as a prop presumably in the future so that the `checked` state can be stored in Redux? Otherwise this component doesn't really serve a purpose yet 🙂

Reply:
We are going to complete redo this page. So no need to review it. It's not slated to be done until 2 sprints from now.

Reply:
This file has been deleted. Because it has no value right now.